README.md

```diff
@@ -1 +1,107 @@
-# apto-pci-sdk-web
\ No newline at end of file
+# AptoWebPCISDK
+
+## Using the Web PCI SDK
+
+AptoWebPCISDK is available as single file to be included in your javascript code
+
+```html
+<script src="https://pci-web.ux.aptopayments.com/apto-pci-sdk.js"></script>
+```
+
+and requires a `div` to load its content into it:
+
+```html
+<div id="apto-pci-sdk"></div>
+```
+
+you can use the `id` that you prefer as long as you provide the right value when calling the `aptoInitialiseSDK` method.
+
+## Initialising the SDK
+
+We recommend initialising the SDK once the page where it will be shown has finished loading:
+
+```html
+<script>
+  function onLoad() {
+    const apiKey = ''
+    const userToken = ''
```

Comment:
We should add a chapter to explain that:

1: apiKey is the Mobile API public key
2: cardId and lastFour have to be obtained previously (Mobile API or Core API)
3: userToken must be obtained using the verifications + login endpoints in the Mobile API (maybe a link to the mobile api specs here?)